dvc/config.py

```diff
@@ -204,6 +204,7 @@ class Config(object):  # pylint: disable=too-many-instance-attributes
     # backward compatibility
     SECTION_GCP = "gcp"
     SECTION_GCP_STORAGEPATH = SECTION_AWS_STORAGEPATH
+    SECTION_GCP_CREDENTIALPATH = "credentialpath"
```

Comment:
In the same that `SECTION_GCP_STORAGEPATH = SECTION_AWS_STORAGEPATH`, we could set `SECTION_GCP_CREDENTIALPATH = SECTION_AWS_CREDENTIALPATH`

Reply:
@mhham Let's do that here as well 🙂